docs/docs/getting-started/04-timer-example.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 4
+---
+
+# A basic application example
+
+Open the Developer Interface and copy the following application code into the Editor.
+
+```yaml
+components:
+  timer:
+    component: base_components::utilities::Timer
+    position:
+      x: 0
+      y: -300
+    parameters:
+      period: 0.1
+      timeout: 2.5
+    events:
+      is_unconfigured:
+        lifecycle:
+          transition: configure
+          component: timer
+      is_inactive:
+        lifecycle:
+          transition: activate
+          component: timer
+      is_timed_out:
+        transition: timer_2
+
+  timer_2:
+    component: base_components::utilities::Timer
+    position:
+      x: 500
+      y: -300
+    parameters:
+      period: 0.1
+      timeout: 2.5
+    events:
+      is_unconfigured:
+        lifecycle:
+          transition: configure
+          component: timer_2
+      is_inactive:
+        lifecycle:
+          transition: activate
+          component: timer_2
+      is_timed_out:
+        transition: timer
+
+on_start:
+  load: timer
+```
+
+The graph should show two components connected with event edges. Press the Play button to start the application.
```

Comment:
Do you think if it looks better if we also include a screenshot of the graph as an example?

Reply:
Screenshots / videos (or gifs) would certainly be nice. I was reluctant to include those at first because they may be outdated sooner than a text description would be, but I guess better to have something than nothing.